includes/Settings.php

```diff
@@ -789,7 +789,7 @@ public function get_locations( $force = false ) {
 
 		$section = isset( $_GET['section'] ) ? sanitize_text_field( wp_unslash( $_GET['section'] ) ) : false;  // phpcs:ignore WordPress.Security.NonceVerification.Recommended
 
-		if ( ! ( $this->is_admin_settings_screen() || ( $this->is_admin_settings_screen() && 'update' !== $section ) || $force ) ) {
+		if ( ! ( $this->is_admin_settings_screen() && ( $this->is_admin_settings_screen() && 'update' !== $section ) || $force ) ) {
```

Comment:
`$this->is_admin_settings_screen()` is checked twice:

```suggestion
		if ( ! (  ( $this->is_admin_settings_screen() && 'update' !== $section ) || $force ) ) {
```